shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/StartScreens.kt

```diff
@@ -0,0 +1,224 @@
+    ) {
+        AnimatedContent(
+            targetState = detailsVisible,
+            modifier = Modifier.weight(1f),
+            transitionSpec = {
+                fadeIn(animationSpec = tween(100, delayMillis = 90))
+                    .togetherWith(fadeOut(animationSpec = tween(90)))
+            }
+        ) { detailsVis ->
+            if (detailsVis) {
+                Column {
+                    MainHeaderTitleBar(
+                        stringResource(AppRes.string.privacy_policy_title),
+                        startContent = {
+                            TopMenuButton(
+                                icon = Res.drawable.arrow_left_24,
+                                contentDescription = stringResource(AppRes.string.privacy_policy_back),
+                                onClick = { detailsVisible = false },
+                            )
+                        }
+                    )
+                    Divider(
+                        thickness = 1.dp,
+                        color = KotlinConfTheme.colors.strokePale,
+                    )
+                    // TODO add actual text of the privacy policy
+                    StyledText("Privacy policy text goes here")
+                    Spacer(Modifier.weight(1f))
+                    Divider(
+                        thickness = 1.dp,
+                        color = KotlinConfTheme.colors.strokePale,
+                    )
+                }
+            } else {
+                Column(
+                    verticalArrangement = Arrangement.spacedBy(24.dp),
+                    modifier = Modifier.padding(horizontal = 12.dp, vertical = 16.dp)
+                ) {
+                    Image(
+                        imageVector = vectorResource(AppRes.drawable.kodee_privacy),
+                        contentDescription = null,
+                        modifier = Modifier.fillMaxWidth()
+                            .size(160.dp)
+                    )
+                    StyledText(
+                        stringResource(AppRes.string.privacy_policy_title),
+                        style = KotlinConfTheme.typography.h1
+                    )
+                    StyledText(
+                        stringResource(AppRes.string.privacy_policy_description),
+                        color = KotlinConfTheme.colors.longText,
+                    )
+                    Action(
+                        stringResource(AppRes.string.privacy_policy_read_action),
+                        icon = Res.drawable.arrow_right_24,
+                        size = ActionSize.Large,
+                        enabled = true,
+                        onClick = { detailsVisible = true }
+                    )
+                }
+                Spacer(Modifier.weight(1f))
+            }
+        }
+
+        Row(
+            horizontalArrangement = Arrangement.spacedBy(16.dp),
+            modifier = Modifier.padding(horizontal = 12.dp, vertical = 16.dp)
```

Comment:
As I started implementing some screens as well, I noticed this `12.dp` padding everywhere.
Should we probably extract it to the theme object?

Reply:
I would only do it if the Figma design can also use it across the entire design, and show it as a named variable. Otherwise it'd be guesswork when this is "the same margin value" and when it isn't.

Reply:
Fair enough